File: sach/de3394.cpp
//DE3394.CPP - Tran Huu Nam - hsgtin.vn - 2024
#include <bits/stdc++.h>
using namespace std;
int n,a,T=0;
bool SNT(int x) {
    if (x<2) return false;
    for (int i=2;i<=sqrt(x);i++)
        if (x%i==0) return false;
    return true;
}
int main () {
  cin>>n;
  for (int i=1;i<=n;i++) {
    cin>>a;     if (SNT(a)) T+= a;
  }
  cout<<T;
}

File: sach/de3394.py
//DE3394.PY - Tran Huu Nam - hsgtin.vn - 2024
import math
def is_prime(x):
    if x < 2:
        return False
    for i in range(2, int(math.sqrt(x)) + 1):
        if x % i == 0:
            return False
    return True
n = int(input())
T = 0
for _ in range(n):
    a = int(input())
    if is_prime(a):
        T += a
print(T)

